helm/values.yaml

```diff
@@ -1207,6 +1239,11 @@ gateway:
 #         type: ClusterIP
 #         externalPort: 18082
 #         appProtocol: http
+    kafka:
+      enabled: true
```

Comment:
Are you sure we want kafka service to be enabled by default ?

Reply:
in fact it's not active because we need gateway.kafka.enable to true too. 
How about adding a comment : 
```
enabled: true # Activated if `gateway.kafka.enabled` are also be enabled
```

the idea of keeping it uncommented and active by default makes it possible to 'activate kafka just with the gateway.kafka. enabled